tests/integration/test_multiple_disks/test.py

```diff
@@ -1120,14 +1126,27 @@ def produce_alter_move(node, name):
 
 
 @pytest.mark.parametrize(
-    "name,engine",
+    "name,engine,use_metadata_cache",
```

Comment:
Looks like you have added a parameter to the wrong test.

Reply:
My mistake. Thanks you for reminding.